doc/manual/parallel-computing.rst

```diff
@@ -562,6 +562,10 @@ arrays, a shared array is accessible only from those participating
 workers specified by the ``pids`` named argument (and the creating
 process too, if it is on the same host).
 
+Note: If you want to store a Type like object, it must be immutable to count
+as a [bitstype](http://docs.julialang.org/en/release-0.4/manual/types/). 
```

Comment:
@Skylion007, to clarify, the wording should be something along the lines of `Note: only objects where `isbits(obj) == true` are supported`. Then link to `docs.julialang.org/en/latest/...` instead of release-0.4. We can the backport the docs change to 0.4.